protocol/consistencychecks.go

```diff
@@ -314,21 +327,15 @@ func (cc *ConsistencyChecks) updateTBs(requestType int, msg *Response,
 	return nil
 }
 
-func (cc *ConsistencyChecks) savePromise(uname string, tb *TemporaryBinding) {
-	cc.TBs[uname] = tb
-	cc.RegEpoch[uname] = cc.SavedSTR.Epoch
-}
-
 // verifyFulfilledPromise verifies issued TBs were inserted
 // in the directory as promised.
 func (cc *ConsistencyChecks) verifyFulfilledPromise(uname string, str *DirSTR,
 	ap *m.AuthenticationPath) error {
-	if regEp, ok := cc.RegEpoch[uname]; ok && str.Epoch == regEp+1 {
```

Comment:
Keeping a `RegEpoch` map for other contacts seems not very nice for me, but yes, this should be fully addressed in a separate PR for #116.